.circleci/config.yml

```
@@ -317,114 +317,40 @@ workflows:
   docker-master-commit:
     jobs:
       - docker-build:
-          name: docker_build_trin
-          target: trin
+          name: docker_build__<< matrix.target >>
```

```suggestion
          name: docker_build_<< matrix.target >>
```

were the double _ intentional?

---

Yes. We already have one in "docker_build" and some targets might contain "_" in the future. I figured that using two would make things clearer.

---

Oh ok :+1: